### PR TITLE
Merge updated documentation to update_fluent191

### PR DESCRIPTION
**Pull request by IshaanDesai**
The updated documentation in [updateDocumentation](https://github.com/precice/fluent-adapter/tree/updateDocumentation) is corresponding to the changes being done in [update_fluent191](https://github.com/precice/fluent-adapter/tree/update_fluent191) To bring the documentation and the corresponding changed code together, this branch is being merged into `update_fluent191`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,104 @@
...
-- set double precision, processing options (serial or parallel) and the dimension
-(for parallel select also "show more", "parallel settings", "mpi types" -> open mpi
+## 2. Installing and Starting Fluent on Ubuntu machine
 
-2.2 How to start Fluent without GUI
-- serial:   fluent 2ddp -g < steer-fluent.txt
-- parallel: fluent 2ddp -g -t4 -mpi=openmpi < steer-fluent.txt
-  (-t4 sets 4 processes for computations)
-  (steer-fluent.txt drives Fluent, can also be done by hand)
+  ### 2.1 How to install Fluent - **ANSYS V19.1 on Linux Ubuntu 16.04 LTS**
```
Comment (BenjaminRodenberg) [inline README.md:28]: You also tried installing Fluent on Ubuntu 18.04, but it did not work. Is this correct? Can you put a comment on this here. You can also link to https://github.com/precice/fluent-adapter/issues/3

File: README.md [hunk 2]
```diff
@@ -1,84 +1,104 @@
...
+      - wave_profile_udf.c: For wave simulations with inflow profile. Needs 
+        wave_profile.c.
+      - wave_maker_udf.c: For wave simulations with moving wall. Needs 
+        wave_maker.c.
+      - fsi_and_wave_profile_udf.c: For FSI + wave simulations. Needs fsi.c and 
+        wave_profile.c.
+
+  * Adapt lnamd64/2ddp_host/user.udf line 3 "FLUENT_INC = ..." to the fluent installation
+    folder which will be of the type *./ansys_inc/v191/fluent*  
+  * Adapt the path of the python library in /src/makefile line 19
```
Comment (BenjaminRodenberg) [inline README.md:16]: Can you give an example here?

File: README.md [hunk 3]
```diff
@@ -1,84 +1,104 @@
...
+  * Update the Ansys RELEASE version in /src/makefile line 26
```
Comment (BenjaminRodenberg) [inline README.md:19]: Can you give an example here? Something like "for FLUENT 19.1 type `VERSION=19.1`. I do not know the actual syntax now, but I hope it is understandable what I mean.

File: README.md [hunk 4]
```diff
@@ -1,84 +1,104 @@
...
+  * Type: make "FLUENT_ARCH=lnamd64" to start the build. Add a "clean" to clean it.
```
Comment (BenjaminRodenberg) [inline README.md]: Put code in ticks for better formatting: `make "FLUENT_ARCH=lnamd64"`

For cleaning, write the full command. Is it something like `make clean "FLUENT_ARCH=lnamd64"` or does `make clean` suffice?

Comment (BenjaminRodenberg) [reply README.md]: Further below, there are some more locations, where ticks should be used for code instructions. For paths we usually also use `code formatting` (e.g. in the wiki), but I am not sure whether there is a convention on this.

**Comment by BenjaminRodenberg** [comment]
I will try to go through the instructions until the end of step **3. Preparing a Fluent .cas file ...** and install Fluent, precice and the adapter following the instructions before our next meeting. Does this suffice as a review or do you need something additional?

**Comment by IshaanDesai** [comment]
> I will try to go through the instructions until the end of step **3. Preparing a Fluent .cas file ...** and install Fluent, precice and the adapter following the instructions before our next meeting. Does this suffice as a review or do you need something additional?

Yes this will suffice the documentation testing, all the steps should finally lead to a properly compiled adapter and a FSI set .cas file for FLUENT use

**Pull request closed without merging**